src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CancelPunchOut/CancelPunchOutCommandHandler.cs

```diff
@@ -51,9 +55,21 @@ private async Task CancelFusionMeetingAsync(Guid meetingId)
             {
                 await _meetingClient.DeleteMeetingAsync(meetingId);
             }
-            catch (Exception e)
+            catch (MeetingApiException e)
             {
-                throw new Exception("Error: Could not cancel outlook meeting.", e);
+                if (e.Code.ToString().ToUpperInvariant() == "FORBIDDEN")
```

Comment:
Is it possible to add an unit test in CancelPunchOutCommandHandlerTests.cs,  proofing that it is the specific MeetingApiException just cause logging and that the handling is OK?

Reply:
Added new test HandlingCancelIpoCommandMeetingApiException_ShouldCallLogErrorOnce